ui/src/components/content/catalog/components/ServiceDetail.tsx

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
would not use the directory name "components" here. May be "services" is a better name.

Reply:
Another suggestion: we can move the Service**.tsx files to the `catalog` , without subdirectory

Reply:
fixed, use services instead of components.